macosx/TorrentTableView.mm

```diff
@@ -221,6 +221,22 @@ - (void)outlineView:(NSOutlineView*)outlineView
             }
         }
     }
+    else
+    {
+        if ([cell isKindOfClass:[GroupTextCell class]])
+        {
+            GroupTextCell* groupCell = cell;
+            groupCell.selected = NO;
+
+            // if cell is selected, set selected flag so we can style the title in GroupTextCell drawInteriorWithFrame
+            NSInteger const row = [self rowForItem:item];
+            NSIndexSet* selectedRowIndexes = self.selectedRowIndexes;
+            if ([selectedRowIndexes containsIndex:row])
+            {
+                groupCell.selected = YES;
+            }
```

Comment:
You're setting `selected` twice (already on line 229).
So best to directly do unconditionally:
```
groupCell.selected = [selectedRowIndexes containsIndex:row];
```

Reply:
done, thanks